src/Forms/Checkout.js

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import Paper from '@material-ui/core/Paper';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import PersonalDetails from "./PersonalDetails";
+import Dates from "./Dates";
+import Payment from "./Payment";
+import SignUpSuccess from "../SignUpSuccess";
+
+class Checkout extends React.Component {
+  state = {
+    activeStep: 0,
+    formControls: {
```

Comment:
It seems to me that `Checkout` should not be concerned with how a form is implemented. I would extract this code and `handleChange/handleBlur` methods into a generic Form component that can handle any type of field.